#!/usr/bin/env python3
PACKAGE = "actor_bumper_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('thresh', int_t, 0, 'Black/White Threshold', 245, 0, 255)

gen.add('blur_factor', int_t, 0, 'Median Blur Factor', 1, 0, 5)

gen.add('crop_l', double_t, 0, "Image Crop Left", 0, 0, 1)
gen.add('crop_r', double_t, 0, "Image Crop Right", 1, 0, 1)
gen.add('crop_t', double_t, 0, "Image Crop Top", 0, 0, 1)
gen.add('crop_b', double_t, 0, "Image Crop Bottom", 1, 0, 1)

gen.add('debug', bool_t, 0, 'Enable Debug Topics', True)





exit(gen.generate(PACKAGE, "bumper_camera", "BumperCam"))